examples/lammps/h2o-committee/init.pdb

```diff
@@ -0,0 +1,651 @@
+TITLE cell{atomic_unit} positions{atomic_unit}
```

Comment:
IMHO we should avoid giving examples with pdb files as inputs. but it is not a big deal. it is just too easy to make a mistake with them and the i-PI parser for pdb is not the most robust in the world.

Reply:
I agree, but pretty much all water examples in lammps/ use this init.pdb. 
I'll open an issue so we can get rid of these (and actually perhaps use a single water.xyz so we save space?

Reply:
Mentioned in #173